wire-schema/src/jvmMain/resources/google/protobuf/descriptor.proto

```diff
@@ -768,6 +789,14 @@ message EnumValueOptions {
   // this is a formalization for deprecating enum values.
   optional bool deprecated = 1 [default = false];
 
+  // Any features defined in the specific edition.
+  optional FeatureSet features = 2;
+
+  // Indicate that fields annotated with this enum value should not be printed
+  // out when using debug formats, e.g. when the field contains sensitive
+  // credentials.
+  optional bool debug_redact = 3 [default = false];
```

Comment:
https://github.com/square/wire/issues/2551

Maybe we keep ignoring it or we ignore the comment saying "when using debug formats" (what is a debug format?)